### PR TITLE
Update compass icon font

### PR DESCRIPTION
**Pull request by deanwhillier**
#### Summary
Updating the Compass Icon font to the [latest version](https://github.com/deanwhillier/mattermost/tree/b045e068fd13c9c35ab5d15e47fb45719468438d) to include new icons needed for ongoing features.

### CODE REVIEW

**Comment by deanwhillier** [comment]
@michaelgamble, @matthewbirtch, looks like two icons are being removed. May just want to double check to be sure that was intended.

File: sass/base/_compass-icons.scss [hunk 1]
```diff
@@ -90,8 +101,8 @@
 .icon-jumbo-attachment-patch:before { content: '\e909'; } /* '' */
 .icon-jumbo-attachment-zip:before { content: '\e90a'; } /* '' */
 .icon-jumbo-attachment-code:before { content: '\e90b'; } /* '' */
+.icon-jumbo-attachment-image-broken:before { content: '\e90c'; } /* '' */
 .icon-account-multiple-outline:before { content: '\f00f'; } /* '' */
-.icon-account-outline:before { content: '\f013'; } /* '' */
```
Comment (matthewbirtch) [inline sass/base/_compass-icons.scss:94]: @michaelgamble is there a reason `icon-account-outline` is being removed?

File: sass/base/_compass-icons.scss [hunk 2]
```diff
@@ -192,7 +203,6 @@
 .icon-bell-off-outline:before { content: '\fa90'; } /* '敖' */
 .icon-chevron-down-circle-outline:before { content: '\fb0c'; } /* '﬌' */
 .icon-format-letter-case:before { content: '\fb19'; } /* '﬙' */
-.icon-account-group-outline:before { content: '\fb34'; } /* 'הּ' */
```
Comment (matthewbirtch) [inline sass/base/_compass-icons.scss:195]: @michaelgamble same with this one. Looks like `icon-account-group-outline` has been removed

**Review by matthewbirtch — COMMENTED** [review]
@michaelgamble as Dean mentioned, two icons seem to have been removed.

**Review by devinbinnie — APPROVED** [review]
LGTM other than Matt's comments

File: sass/base/_compass-icons.scss (hunk 1 above)
Comment (deanwhillier) [reply sass/base/_compass-icons.scss:94]: Actually looks like this one just moved further down ... git is not always the best at picking changes up with they are moved. See ln. 217 on the right side.

File: sass/base/_compass-icons.scss (hunk 2 above)
Comment (deanwhillier) [reply sass/base/_compass-icons.scss:195]: Looks like this one never come over ... I checked the svg folder on my initial commit and it's not there. Maybe I/we forgot to add it in here?

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1: Thanks @deanwhillier!

File: sass/base/_compass-icons.scss (hunk 2 above)
Comment (matthewbirtch) [reply sass/base/_compass-icons.scss:195]: Maybe. That would be strange though since it's in our original files.

Comment (matthewbirtch) [reply sass/base/_compass-icons.scss:195]: Actually, I'm wrong. `icon-account-group-outline` is not part of our original files after all. I'm not sure how it was in there in the first place.

**Review by matthewbirtch — APPROVED** [review]

**Comment by deanwhillier** [comment]
@matthewbirtch, the additional icons have been added, please double check when you have a minute.

**Review by matthewbirtch — APPROVED** [review]
looks good to me @deanwhillier

**Review by jgilliam17 — APPROVED** [review]
Thank you @deanwhillier 
Tested, looks good to merge.
- Performed a quick check on the existing icons as updated ones aren't in use yet in the app; existing icons are showing up properly on the test server.

**Pull request merged**